playbooks/roles/cloudflare/tasks/main.yml

```diff
@@ -1,6 +1,10 @@
 ---
 - name: load vars
   local_action: include_vars conf.yml
+  when:
+    (cf_zone is undefined) or
+    (cf_token is undefined) or
+    (cf_zone is undefined)
 
```

Comment:
Only two comments:
- cf_zone is duplicated, please include it once only. You should also include main and subdomain in this check, right?
- can we have CF_ZONE as an environment variable?